python/paddle/fluid/tests/unittests/test_imperative_LodtensorToSelectedrows.py

```diff
@@ -0,0 +1,192 @@
+                        is_sparse=is_sparse)
+
+                    sgd = SGDOptimizer(learning_rate=1e-3)
+                    dy_param_updated = dict()
+                    dy_param_init = dict()
+                    dy_loss = None
+
+                    helper = DyGraphProgramDescTracerTestHelper(self)
+                    backward_strategy = fluid.dygraph.BackwardStrategy()
+                    backward_strategy.sort_sum_gradient = is_sort_sum_gradient
+
+                    for i in range(batch_num):
+                        x_data = np.arange(12).reshape(4, 3).astype('int64')
+                        y_data = np.arange(1, 13).reshape(4, 3).astype('int64')
+                        x_data = x_data.reshape((-1, num_steps, 1))
+                        y_data = y_data.reshape((-1, 1))
+
+                        x = to_variable(x_data)
+                        y = to_variable(y_data)
+                        outs = simple_net(x, y)
+                        dy_loss = outs
+                        if i == 0:
+                            for param in simple_net.parameters():
+                                dy_param_init[param.name] = param.numpy()
+                        dy_loss.backward(backward_strategy)
+                        sgd.minimize(dy_loss)
+                        simple_net.clear_gradients()
+                        if i == batch_num - 1:
+                            for param in simple_net.parameters():
+                                dy_param_updated[param.name] = param.numpy()
+                    dy_loss_value = dy_loss.numpy()
+
+                with new_program_scope():
+                    fluid.default_startup_program().random_seed = seed
+                    fluid.default_main_program().random_seed = seed
+
+                    simple_net = SimpleNet(
+                        "simple_net",
+                        hidden_size=hidden_size,
+                        vocab_size=vocab_size,
+                        num_steps=num_steps,
+                        is_sparse=is_sparse)
+
+                    exe = fluid.Executor(fluid.CPUPlace()
+                                         if not core.is_compiled_with_cuda()
+                                         else fluid.CUDAPlace(0))
+                    sgd = SGDOptimizer(learning_rate=1e-3)
+                    x = fluid.layers.data(
+                        name="x", shape=[-1, num_steps, 1], dtype='int64')
+                    y = fluid.layers.data(
+                        name="y", shape=[-1, 1], dtype='float32')
+
+                    static_loss = simple_net(x, y)
+                    sgd.minimize(static_loss)
+                    static_param_updated = dict()
+                    static_param_init = dict()
+                    static_param_name_list = list()
+                    for param in simple_net.parameters():
+                        static_param_name_list.append(param.name)
+
+                    out = exe.run(framework.default_startup_program(),
```

Comment:
why not use `fluid.default_startup_program`

Reply:
done